index.html

```diff
@@ -0,0 +1,346 @@
+              <li>CSS</li>
+            </ul>
+          </div>
+        </li>
+
+        <li>
+          <div class="type-of-cards">
+            <div class="cat-images">
+              <img src="frame.svg" alt="picture of globe" />
+            </div>
+
+            <h2 class="cat-title">Frameworks</h2>
+            <div>
+              <ul class="cat-things">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+              </ul>
+            </div>
+          </div>
+        </li>
+
+        <li>
+          <div class="type-of-cards">
+            <div class="cat-images">
+              <img src="ball.svg" alt="picture of globe" />
+            </div>
+
+            <h2 class="cat-title">Skills</h2>
+
+            <ul class="cat-things">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </li>
+      </ul>
+    </section>
+
+    <footer class="contactpage">
+      <section class="projects-end">
+        <h2 class="textline1">I'm always</h2>
+        <h3 class="textline2">interested in</h3>
+        <h4 class="textline3">hearing about</h4>
+        <h5 class="textline4">new projects, so</h5>
+        <h6 class="textline5">if you'd like to</h6>
+        <h6 class="textline6">chat please get</h6>
+        <h6 class="textline7">in touch</h6>
+      </section>
+      <section class="projects-2-big">
+        <div class="new-paragraph">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </div>
+      </section>
+
+      <section class="form-section">
+        <form action="https://formspree.io/f/myyvzkag" method="post">
+          <div class="input-box-box">
+            <ul class="form-info">
+              <li>
+                <input
+                  type="text"
+                  maxlength="30"
+                  name="user_name"
+                  class="name-text-box"
+                  id="full-name"
+                  placeholder="Full Name"
+                  required=""
+                />
+              </li>
+
+              <li>
+                <input
+                  type="email"
+                  name="user_email"
+                  class="name-last-text-box"
+                  id="email"
+                  placeholder="Email Address"
+                  required=""
+                />
+              </li>
+              <li>
+                <textarea
+                  id="text-box"
+                  name="message"
+                  maxlength="500"
+                  class="enter-form"
+                  cols="30"
+                  rows="10"
+                  placeholder="Enter text here"
+                  required=""
+                ></textarea>
```

Comment:
- [ ] great job, your form works and submits successfully. Let's work a little on the design to perfect it. Kindly remove the black border from the submit button
- [ ] please add the extra space below the contact form like in the Figma design
- [ ] kindly add the margin top/bottom of your inputs, and please make the placeholder text a little bit bigger 
your design 
![Screenshot (369)_LI](https://user-images.githubusercontent.com/65298048/187549229-434cf842-bc74-448f-bfe3-327b2811128b.jpg)
figma
![Screenshot (369)](https://user-images.githubusercontent.com/65298048/187549400-45bb34d2-8d5d-484b-baeb-47b8ea6510e5.png)